bin/cron/create_rollup_tables

```diff
@@ -3,11 +3,10 @@
 # This script populates (after truncating) the contained_levels,
 # contained_level_answers, and the level_sources_multi_type tables.
 
+require_relative '../../dashboard/config/environment'
 require 'cdo/only_one'
 exit unless only_one_running?(__FILE__)
```

Comment:
While we're in here, I think we should [follow Will's lead](https://github.com/code-dot-org/code-dot-org/pull/35287) and convert the `only_one` require into a `require_relative`, instead of doing the slow environment load before this check.

Reply:
✅